pkg/noderesourcetopology/filter.go

```diff
@@ -44,23 +46,46 @@ func singleNUMAContainerLevelHandler(pod *v1.Pod, zones topologyv1alpha1.ZoneLis
 	// Node() != nil already verified in Filter(), which is the only public entry point
 	logNumaNodes("container handler NUMA resources", nodeInfo.Node().Name, nodes)
 
-	// We count here in the way TopologyManager is doing it, IOW we put InitContainers
-	// and normal containers in the one scope
-	for _, container := range append(pod.Spec.InitContainers, pod.Spec.Containers...) {
+	// the init containers are running SERIALLY and BEFORE the normal containers
```

Comment:
add a reference (link) to https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#understanding-init-containers

Reply:
Done.